components/fxa-client/src/http_client.rs

```diff
@@ -472,6 +526,10 @@ impl AuthorizationParameters {
     }
 }
 
+// Keeping those functions out of the FxAClient trate becouse functions in the
```

Comment:
Added those because of the Client::new() 😭 Tried adding the functions into the FxAClient trait, but since they are test only they made the mockiato upset and unable to compile tests. Any ideas, or would them hanging around here be okay?

Reply:
That's fine for now, this comment here will be helpful for the future 👍